azure-webapp-maven-plugin/v1-schema.md

```diff
@@ -0,0 +1,128 @@
+## Webapp configuration V1 Schema(Deprecated)
+
+#### Config
+
+Property | Required | Description | Version
+---|---|---|---
+`<region>`* | false | Specifies the region where your Web App will be hosted; the default value is **westeurope**. All valid regions at [Supported Regions](#region) section. | 0.1.0+
+`<resourceGroup>` | true | Azure Resource Group for your Web App. | 0.1.0+
+`<appName>` | true | The name of your Web App. | 0.1.0+
+`<pricingTier>`* | false | The pricing tier for your Web App. The default value is **P1V2**.| 0.1.0+
+`<deploymentSlot>` | false | The deployment slot to deploy your application. | 1.3.0+
+`<appServicePlanResourceGroup>` | false | The resource group of the existing App Service Plan. If not specified, the value defined in `<resourceGroup>` will be used by default. | 1.0.0+
+`<appServicePlanName>` | false | The name of the existing App Service Plan. | 1.0.0+
+`<appSettings>` | false | Specifies the application settings for your Web App. | 0.1.0+
+`<stopAppDuringDeployment>` | false | To stop the target Web App or not during deployment. This will prevent deployment failure caused by IIS locking files. | 0.1.4+
+  
+
+#### Runtime settings
+
+  Details about the supported values could be found [here](README.md). Tomcat 8.5 will be used as default value for `<javaWebContainer>`.
+  
+- **Web App on Linux**
+  ```xml
+  <configuration>
+  ...
+    <linuxRuntime>tomcat 8.5-jre8</linuxRuntime>
+  </configuration>
+  ```
+  The supported values are *tomcat 8.5-jre8*, *tomcat 9.0-jre8*, *wildfly 14-jre8*, and *jre8*.
+
+- **Web App on Windows**
+  ```xml
+  <configuration>
+  ...
+    <javaVersion>1.8</javaVersion>
+    <javaWebContainer>tomcat 8.5</javaWebContainer>
+  </configuration>
+  ```
+
+- **Web App for Containers**    
+  ```xml
+  <configuration>
+  ...
+    <containerSettings>
+    <!-- only the imageName is required --> 
+      <imageName>[hub-user/]repo-name[:tag]</imageName>
+      <serverId></serverId>
+      <registryUrl></registryUrl>
+    </containerSettings>
+  </configuration>
+  ```
+
+    `tag` is optional for the `imageName`. The default value is latest. Public docker hubs and private container registries are both supported.
+
+
+#### Deployment settings
+
+  There are multiple deployment types are supported:
```

Comment:
I remember there is another deployment type called 'NONE'?

Reply:
There is a NONE type, should add to doc